packages/components/psammead-timestamp/README.md

```diff
@@ -0,0 +1,51 @@
+# psammead-timestamp
```

Comment:
This will need the various badges we use, an example can be seen in https://github.com/bbc/psammead/blob/latest/packages/components/psammead-brand/README.md